hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/policy/DiskBalancerVolumeChoosingPolicy.java

```diff
@@ -30,11 +30,11 @@ public interface DiskBalancerVolumeChoosingPolicy {
    * Choose a pair of volumes for balancing.
    *
    * @param volumeSet - volumes to choose from.
-   * @param threshold - the threshold to choose source and dest volumes.
+   * @param thresholdPercentage the threshold percentage in range [0, 100] to choose the source volume.
```

Comment:
Threshold range 0 and 100 are exclusive, so (0, 100).

Reply:
Ya, "threshold.percentage" is more clear, and a overall check of threshold configuration is desired.